app.js

```diff
@@ -2,21 +2,31 @@ const createError = require('http-errors');
 const express = require('express');
 const cookieParser = require('cookie-parser');
 const logger = require('./src/middlewares/logger.middelware');
+const routes = require('./src/routes/routes');
+const mongoose = require('mongoose');
 
 const app = express();
 
+// connect to db
+if (process.env.NODE_ENV !== 'test') {
+    mongoose.connect('mongodb://localhost/aut_graduation');
+}
+
+// view engine setup
```

Comment:
If you can please delete this line of comment cause we don't have any view engine, it is most certainly a mistake by me not you, but please fix it.

Reply:
Done.